app/lib/config/config.go

```diff
@@ -237,3 +240,13 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
+	if strings.HasPrefix(path, "~") {
+		usr, err := user.Current()
+		ui.ErrFatal(err)
```

Comment:
Moreover, did we not remove that ui library ? Just looked in my local repo, it seems both are duplicated ?